tests/components/unifi/test_device_tracker.py

```diff
@@ -217,10 +226,9 @@ async def test_tracked_clients(hass):
         hass,
         options={CONF_SSID_FILTER: ["ssid"]},
         clients_response=[CLIENT_1, CLIENT_2, CLIENT_3, CLIENT_5, client_4_copy],
-        devices_response=[DEVICE_1, DEVICE_2],
-        known_wireless_clients=([CLIENT_4["mac"]]),
+        known_wireless_clients=(CLIENT_4["mac"],),
     )
-    assert len(hass.states.async_entity_ids(TRACKER_DOMAIN)) == 6
+    assert len(hass.states.async_entity_ids(TRACKER_DOMAIN)) == 4
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)